docs/contribute/README.md

```diff
@@ -78,7 +78,7 @@ flowforge-dev-env
 More details on using the FlowForge Development Environment are available in its
```

Comment:
This still refers to `FlowForge Development Environment` however the link to the developer env on github (up at line 49) was renamed to `[FlowForge Development Environment](https://github.com/flowforge/flowforge-dev-env)`

Reply:
Good catch